<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>georgetown1</class>
 <widget class="QDialog" name="georgetown1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>490</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="displayName">
      <property name="text">
       <string>Display Table</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="text">
       <string>Update Records</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonagregar">
      <property name="text">
       <string>Add Team</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="text">
       <string>Delete Team</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>861</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QTableWidget" name="tableDatos">
      <column>
       <property name="text">
        <string>Team</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Games</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Win</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Loss</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tied</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dif</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Pts</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Team</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditnombre"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Games</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Win</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditwin"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Tied</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdittied"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Loss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditLoss"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Dif</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDiff"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Pts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditpts"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
